zerver/openapi/zulip.yaml

```diff
@@ -1288,7 +1331,7 @@ paths:
                     },
                     {
                         "id":6,
-                        "value":"https:\/\/zulipchat.com"
+                        "value":"https://zulipchat.com"
```

Comment:
Have you inspected how the logs look?  It's possible we need those backslashes.

Reply:
Its in the examples part so I think it must have been a typo by rhe creator.